tools/wptrunner/wptrunner/wptrunner.py

```diff
@@ -150,6 +152,133 @@ def get_pause_after_test(test_loader, **kwargs):
     return kwargs["pause_after_test"]
 
 
+def run_test_iteration(counts, test_loader, test_source_kwargs,
+                       test_source_cls, run_info, recording,
+                       test_environment, product, run_test_kwargs):
+    """Runs the entire test suite.
+    This is called for each repeat run requested."""
+    tests = []
+    for test_type in test_loader.test_types:
+        tests.extend(test_loader.tests[test_type])
+
+    try:
+        test_groups = test_source_cls.tests_by_group(
+            tests, **test_source_kwargs)
+    except Exception:
+        logger.critical("Loading tests failed")
+        return False
+
+    logger.suite_start(test_groups,
+                       name='web-platform-test',
+                       run_info=run_info,
+                       extra={"run_by_dir": run_test_kwargs["run_by_dir"]})
+    for test_type in run_test_kwargs["test_types"]:
+        logger.info("Running %s tests" % test_type)
+
+        browser_cls = product.get_browser_cls(test_type)
+
+        browser_kwargs = \
```

Comment:
So a bunch of the reformatting here feels unnecessary, and this use of a continuation line in particular is not in keeping with the general project style.

Reply:
I'll remove all instances of these line continuations - noted to not use these in the future!